### ISSUE TITLE
cssProp isn't working with parcel

### ISSUE THREAD

**Issue by nathanspecht** [styled-components/babel-plugin-styled-components#203]
## Description

The cssProp feature for babel-plugin-styled-components does not work with Parcel.

## Error

```
Invariant Violation
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `App`.
```

## Dependencies

```
"babel-plugin-styled-components": "1.10.0"
"react": "16.4.2"
"react-dom": "16.4.2"
"styled-components": "4.1.3"
"@babel/cli": "^7.0.0"
"@babel/core": "^7.0.0"
"@babel/plugin-proposal-class-properties": "^7.0.0"
"@babel/preset-env": "^7.0.0"
"babel-core": "7.0.0-bridge.0"
"babel-plugin-transform-es2015-modules-commonjs": "^6.26.2"
"babel-minify": "^0.4.3"
"babel-plugin-transform-class-properties": "^6.24.1"
"babel-preset-es2015": "^6.24.1"
"babel-preset-react": "^6.24.1"
"babel-preset-stage-3": "^6.24.1"
"parcel-bundler": "^1.6.1"
```

## .babelrc

```
{
  "presets": [
    "@babel/preset-env",
    "@babel/preset-react"
  ],
  "plugins": [
    ["babel-plugin-styled-components", {
      "cssProp": true
    }],
    ["@babel/plugin-proposal-class-properties", { "loose": true }]
  ]
}
```

## CodeSandbox

https://codesandbox.io/s/j727ky0p55

**Comment by LanhKhoc** [comment]
Btw, it doesn't work with **@babel/plugin-transform-react-constant-elements** too.

![image](https://user-images.githubusercontent.com/35893600/53547898-ce91b480-3b62-11e9-9b9f-b4d0501d6b74.png)

**Comment by probablyup** [comment]
Hmm I think this is because of injection order... have some ideas on how to fix it.

**Comment by probablyup** [comment]
Ok yeah I've got a fix for this in #228. Example sandbox using a test build: https://codesandbox.io/s/react-parcel-starter-o2wov?fontsize=14

**Comment by probablyup** [comment]
@LanhKhoc please make a separate issue for that with a reproduction sandbox :)

**Comment by probablyup** [comment]
Closed via #228 - it'll go out with the next patch release or you can try it now with: `babel-plugin-styled-components@ej-test`
